src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -2900,6 +2900,19 @@ void GraphOptimizer::MergeReorderAndTranspose(Graph &graph) {
         auto& outOrder = outBlockedDesc->getOrder();
 
         if (checkAscendingFinalOrder(transposeOrder, layoutOrder, inOrder, outOrder)) {
+            const auto parentInPlace = parentNode->getParentEdgeAt(0)->inPlace(Edge::LOOK_UP);
+            const auto& childEdges = childNode->getChildEdgesAtPort(0);
+            const auto childInPlace = std::any_of(childEdges.begin(), childEdges.end(), [](const EdgePtr& edge) {
+                return edge->inPlace(Edge::LOOK_DOWN);
+            });
```

Comment:
I see the same code is executed in `mergeTransposeReshapeReorder`. Sound like it should be extracted into separate helper function with some meaningful name like `canBeInplaced`.

Reply:
Thanks for the suggestion, applied